GitUtils.ps1

```diff
@@ -313,7 +313,13 @@ function Start-SshAgent([switch]$Quiet) {
             }
         }
     }
-    Add-SshKey
+    $keyfiles = Get-ChildItem -Path "$home/.ssh/*" -Exclude *.*, known_hosts

```

Comment:
Maybe extract `Get-SshDirectory` from `Get-SshPath` so we're consistent in referencing that path? Just noticed we have slightly different logic in the `plink` branch of `Add-SshKey`.